src/main/jbake/content/blog/2016/10/25/hawkular-metrics-0.21.0.Final-released.md

```diff
@@ -0,0 +1,63 @@
+title=Hawkular Metrics 0.21.0 - Release
+author=Stefan Negrea
+date=2016-10-25
+type=post
+tags=blog, metrics, release
+status=published
+~~~~~~
+
+I am happy to announce release 0.21.0 of Hawkular Metrics.This release is anchored by performance enhancements and general fixes.
+
+Here is a list of major changes:
+
+* Cassandra
+    * **Cassandra 3.0.9 is now the supported version of Cassandra.**
+    * Note: this is a rollback from previously supported version of 3.7 due to Cassandra community recommendations for stability and production deployment. Cassandra 3.7 or 3.9 are still compatible but development and testing now use 3.0.9 release.
```

Comment:
3.9 works with Hawkular Metrics, we briefly tested it when we had to make the decision to move from 3.7.